docs/general/adapters.md

```diff
@@ -194,6 +194,43 @@ The user could pass in `include=**`.
 
 We recommend filtering any user-supplied includes appropriately.
 
+#### Include Parameters
+
+You may wish to include related resources based on the `include` request
+parameter (See [fetching_includes](http://jsonapi.org/format/#fetching-includes). You can opt-in to this pattern:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  associations_via_include_param(true)
```

Comment:
The problem this solves is having to take a DB hit to render `id/types`, when `id/types` are not needed in the response. Instead, we render only the `links`, or nothing at all. Whatever code gets that done, :+1: